system/BaseModel.php

```diff
@@ -198,7 +197,7 @@ abstract class BaseModel
     /**
      * Our validator instance.
      *
-     * @var Validation
+     * @var ValidationInterface
      */
     protected $validation;
```

Comment:
It seems to me that it makes sense to specify the property type directly instead of docBlock in BaseModel and Controller.

Reply:
The existing user's Model will break.
https://3v4l.org/NkQVr

Reply:
Why override the property?

Reply:
Good question. I don't know, but we can override.
I should have said, the existing user's Model might break.